adminSiteClient/EditorBasicTab.tsx

```diff
@@ -87,7 +87,10 @@ class DimensionSlotView extends React.Component<{
 
         if (grapher.isScatter || grapher.isSlopeChart || grapher.isMarimekko) {
             selection.clearSelection()
-        } else if (grapher.yColumnsFromDimensions.length > 1) {
+        } else if (
+            grapher.yColumnsFromDimensions.length <= 1 &&
+            !grapher.isStackedArea
```

Comment:
Not sure if this is intended; if it is, could you quickly explain why you made that change? I don't immediately see what it could be for.

Reply:
This is to make sure that multi-entity selection is the default for all stacked area charts.

The gist of the PR is basically: Whenever the author selects Stacked Area from the chart type dropdown, make sure multi-entity selection is enabled automatically.

Reply:
Right, the `!grapher.isStackedArea` change I get.
But what is changing the `> 1` to `<= 1` for? I don't think we want that.